src/avk.cpp

```diff
@@ -3848,6 +3848,25 @@ namespace avk
 		);
 	}
 
+	framebuffer root::create_framebuffer_from_template(resource_reference<const framebuffer_t> aTemplate, std::function<void(image_t&)> aAlterImageConfigBeforeCreation,
+		std::function<void(image_view_t&)> aAlterImageViewConfigBeforeCreation, std::function<void(framebuffer_t&)> aAlterFramebufferConfigBeforeCreation)
+	{
+		const auto& templateImageViews = aTemplate->image_views();
+		std::vector<resource_ownership<avk::image_view_t>> imageViews;
+
+		for (auto& imView : templateImageViews)
+		{
+			auto imageView = create_image_view_from_template(imView, aAlterImageConfigBeforeCreation, aAlterImageViewConfigBeforeCreation);
+			imageViews.emplace_back(std::move(imageView));
+		}	
+
+		return create_framebuffer(

```

Comment:
Actually, in order to be 100% correct, the following `create_framebuffer` overload should be used:
```
framebuffer root::create_framebuffer(resource_ownership<renderpass_t> aRenderpass, std::vector<resource_ownership<image_view_t>> aImageViews, uint32_t aWidth, uint32_t aHeight, std::function<void(framebuffer_t&)> aAlterConfigBeforeCreation)
```
i.e. the one that has the `uint32_t aWidth, uint32_t aHeight` parameters. This one is the most specific of the overloads and is the one which is used to _actually_ create a framebuffer. I.e. you'd have to read with and height from the old framebuffer and pass those two values to `create_framebuffer`!
(I'm not 100% sure if it really has some real-world usages... but it would be the correct way w.r.t. the current state of _Auto-Vk_.)